app/src/main/java/grinnell/appdev/edu/lyles/ItemAdapter.java

```diff
@@ -0,0 +1,64 @@
+package grinnell.appdev.edu.lyles;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+
+import grinnell.appdev.edu.lyles.preferences.FavoritesManager;
+
+/**
+ * Created by Shelby on 2/4/2017.
+ */
+
+public class ItemAdapter extends ArrayAdapter<MenuItem> {
+
+    private FavoritesManager favoritesManager;
+    private boolean isFavTab;
```

Comment:
I don't think this variable name is really descriptive. We can probably use something like `isFavoriteTabClicked`